src/platforms/shared/bidders/bidders-state.setup.ts

```diff
@@ -26,6 +26,7 @@ export class BiddersStateSetup implements DiProcess {
 		triplelift: 'icPrebidTriplelift',
 		verizon: 'icPrebidVerizon',
 		yahoossp: 'icPrebidYahooSsp',
+		relevantdigital: 'icWebAds',
```

Comment:
Shouldn't this be like that?
```suggestion
		relevantdigital: 'icPrebidWebAds',
```
Also, why `relevantdigital`?

Reply:
It's the name in Prebid library, we use the name WebAds for our purpose.

Reply:
What about the suggested change? Just want to make sure you didn't miss it.